app/src/main/res/layout/quick_keyboard_button.xml

```diff
@@ -12,16 +12,14 @@
 <ImageView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I don't know how you feel about it, but the TextAction-button worked really well & flawless since long time. I don't see the need to change something there. Do you?

Reply:
There is an issue with how variable padding interacts with the layout. This is an effort to correct it and make the button spacing change cleanly. But all of this is still experimental. I will look at it later, after snippets is done.